Tests/designspaceLib/data/test_avar2.designspace

```diff
@@ -0,0 +1,1481 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="jstf" name="Justify" minimum="-100" maximum="100" default="0"/>
```

Comment:
where is this lowercase "jstf" axis (impliing an official registered axis) defined? I don't see it in 
https://learn.microsoft.com/en-us/typography/opentype/spec/dvaraxisreg#registered-axis-tags

suggest making this uppercase JSTF until it's officially defined.

By the way, why a justification axis to influence the weight? Is that a realistic scenario -- just curious

Reply:
+1 width